src/optionsStorage.ts

```
@@ -170,6 +181,11 @@ export const disabledSettings = proxy({
 })
 
 const migrateOptions = (options: Partial<AppOptions & Record<string, any>>) => {
+  if (options.dayCycleAndLighting) {
+    delete options.dayCycleAndLighting
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)